fastembed/text/onnx_embedding.py

```diff
@@ -98,6 +98,15 @@
             "hf": "qdrant/all-MiniLM-L6-v2-onnx",
         },
     },
+    {
+        "model": "nomic-ai/nomic-embed-text-v1",
+        "dim": 768,
+        "description": "8192 context length english model",
+        "size_in_GB": 0.54,
+        "sources": {
+            "hf": "xenova/nomic-embed-text-v1",
```

Comment:
@NirantK, do we use nomic-ai/nomic-embed-text-v1 as the source instead? Has Onnx weights and the required structure.